### PR TITLE
feat(titus): Render run job output file as YAML

### PR DESCRIPTION
**Pull request by christopherthielen**
Script stage and Titus Run Job stage can return JSON, Properties file, or Yaml.   This PR introduces a component to render all three as YAML and inside a `<pre>` tag.  It uses the component to render the output file of the Script and Titus Run Job stages.

Key/Value pairs (i.e., properties files) are rendered as YAML.
Screen Shot 2019-05-13 at 12 40 34 PM

It also supports nested objects (as seen in JSON/YAML)
Screen Shot 2019-05-13 at 12 34 47 PM

Additionally, it linkifies any URLs found as previously seen in the Run Job stage.
Screen Shot 2019-05-14 at 3 23 09 PM

### CODE REVIEW

File: app/scripts/modules/core/src/presentation/RenderOutputFile.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { dump as dumpYaml } from 'js-yaml';
+
+interface IRenderOutputFileProps {
+  outputFileObject: object;
+  options?: {
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/presentation/RenderOutputFile.tsx]: should we omit the options? we only ever set them to `{ transformLinks: true }`

Comment (christopherthielen) [reply app/scripts/modules/core/src/presentation/RenderOutputFile.tsx]: if you think its a good idea

Comment (christopherthielen) [reply app/scripts/modules/core/src/presentation/RenderOutputFile.tsx]: it would certainly save 6 or 7 lines of code that we really can't afford

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/presentation/RenderOutputFile.tsx]: it'll be more performant without all this cruft

**Pull request merged**